buildengine/deps.go

```diff
@@ -89,6 +93,41 @@ func extractGoFTLImports(self, dir string) ([]string, error) {
 	return modules, nil
 }
 
-func extractKotlinFTLImports(dir string) ([]string, error) {
-	panic("not implemented")
+func extractKotlinFTLImports(self, dir string) ([]string, error) {
+	dependencies := map[string]bool{}
+	kotlinImportRegex := regexp.MustCompile(`^import ftl\.([A-Za-z0-9_.]+)`)
+
+	err := filepath.WalkDir(dir, func(path string, d fs.DirEntry, err error) error {
```

Comment:
I was thinking last night that this could probably just search `filepath.Join(dir, "src/main/kotlin")`?